controllers/room.js

```diff
@@ -0,0 +1,42 @@
+const RoomService = require('../services/room');
+
+class RoomController {
+    static create = async(req,res)=>  {
+        const {name, type, max_people_inside} = req.body;
+        const response = await RoomService.createRoom({ name, type, max_people_inside });
+        return res.status(200).json(response);
```

Comment:
Verify room with this name doesn't exist

Reply:
Done